repo/gromacs/package.py

```diff
@@ -539,13 +539,20 @@ def cmake_args(self):
         if "+cuda" in self.spec:
             options.append("-DCUDA_TOOLKIT_ROOT_DIR:STRING=" + self.spec["cuda"].prefix)
 
-        options.append("-DGMX_EXTERNAL_LAPACK:BOOL=ON")
-        if self.spec["lapack"].libs:
-            options.append("-DGMX_LAPACK_USER={0}".format(self.spec["lapack"].libs.joined(";")))
+        target = self.spec.target
+        if "+cuda" in self.spec and target.family == "ppc64le":
+            options.append("-DGMX_EXTERNAL_LAPACK:BOOL=OFF")
```

Comment:
This is to disable the use of external lapack on a specific system if I understand: `LLNL-Sierra-IBM-power9-V100-Infiniband/spack.yaml`. I think it would be useful to add an explicit comment about that so it's easier to correct in the future.